com.civicrm.hrjobroles/js/src/job-roles/controllers/add-new-role.html

```diff
@@ -225,23 +236,22 @@
                 <tr ng-repeat="(key, funder) in jobroles.editData['new_role_id']['funders']">
                   <td>
                     <!--show funder in non-edit mode-->
-                    <span ng-show="!tableform.$visible">{{funder.funder_id.sort_name}}</span>
+                    <span ng-show="!tableform.$visible">{{funder.funder_id.title}}</span>
                     <!--show ui-select for funder in edit mode-->
                     <span ng-show="tableform.$visible">
-                      <ui-select class="funder-select"
+                      <ui-select
+                        class="funder-select"
                         prevent-animations
                         theme="civihr-ui-select"
                         ng-model="funder.funder_id">
                         <ui-select-match prevent-animations allow-clear placeholder="Select a funder" class="ui-select-match">
-                          {{$select.selected.sort_name}}
+                          {{$select.selected.value.title}}
                         </ui-select-match>
                         <ui-select-choices
                           prevent-animations
                           class="ui-select-choices"
-                          repeat="contact in jobroles.contactList | filter: $select.search"
-                          refresh="jobroles.getContactList($select.search)"
-                          refresh-delay="500">
-                          <div ng-bind-html="contact.sort_name | highlight: $select.search"></div>
+                          repeat="funder.value.value as (key,funder) in jobroles.FundersData | filter: { value: { title: $select.search }}">
```

Comment:
`funder.value.value` just looks a bit odd. Could you please explain what is `funder.value` and what is `funder.value.value`? I have a feeling that `funder.value` should be `funder.properties` or smth.

Reply:
Shouldn't it be then smth like `funder.properties.value`?

Reply:
right. let's leave it like it is now.